### PR TITLE
Prl 3529 courtnav fixes

### PR DESCRIPTION
**Pull request by swaroopa86**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [x] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-3529

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/private-law/json/FixedLists.json [hunk 1]
```diff
@@ -4324,29 +4324,29 @@
   {
     "LiveFrom": "01/12/2022",
     "ID": "DioBeforeAEnum",
-    "ListElementCode": "legalAdviser",
-    "ListElement": "Legal adviser",
+    "ListElementCode": "circuitJudge",
+    "ListElement": "Circuit judge",
```
Comment (AlokDatta) [inline definitions/private-law/json/FixedLists.json:4328]: why this change? any change in this file?

Comment (swaroopa86) [reply definitions/private-law/json/FixedLists.json:4328]: this changes has been reverted

**Review by shashisk97 — APPROVED** [review]
LGTM
